src/main/resources/application.yml

```diff
@@ -43,6 +43,8 @@ thread-pool:
     max-size: 5
 
 logging:
+  pattern:
+    level: "%X{correlationId}%5p"
```

Comment:
Not sure why the property is logging.pattern.level ... but is is!
This results in log statements like this, where the correlation ID comes straight after the date:
```
2022-11-18 13:58:40.726 1b9e3569caad4a4fab8538f61c14c6a7 INFO 50873 --- [enerContainer-5] .m.SendApplicationToPrintMessageListener : Print message with source reference [63778f8fe83c0a36ba274284] saved
```